src/main/java/com/easypost/service/ReferralCustomerService.java

```diff
@@ -220,7 +222,8 @@ private PaymentMethodObject createEasypostCreditCard(final String referralApiKey
         wrappedParams.put("credit_card", params);
 
         EasyPostClient referralClient = new EasyPostClient(referralApiKey);
-        return Requestor.request(RequestMethod.POST, String.format("%s/%s", EasyPost.API_BASE, "credit_cards"),
+        return Requestor.request(RequestMethod.POST,
+                String.format("%s/%s/%s", client.getApiBase(), client.getApiVersion(), "credit_cards"),
```

Comment:
I'd prefer we set all these `String.format` items to an intermediary variable so we know what it is and these lines can be much shorter. Thoughts?

If you agree, we should do that throughout the whole project. It just seems crammed with so much text on one line which then wraps to two or three..

Reply:
I agree, this is resolved.